applications/climatology/climatology.ipynb

```diff
@@ -0,0 +1,3533 @@
+{
```

Comment:
<p>I think it would make more sense to have the "xarray user guide" link point to <a href="https://xarray.pydata.org/en/v2024.06.0/user-guide/time-series.html#datetime-components" target="_blank">https://xarray.pydata.org/en/v2024.06.0/user-guide/time-series.html#datetime-components</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>